src/configs/redisConfigs.js

```diff
@@ -0,0 +1,26 @@
+import redis from 'redis';
+import dotenv from 'dotenv';
+import pino from 'pino';
+
+dotenv.config();
+
+const consoleLog = pino();
```

Comment:
@MCFrank16 what does pino do?

Reply:
As it is not allowed to console log out some of the messages, I just used PINO to help me in knowing whether the Redis connection is well established or not. so PINO consoles out the message